pkg/reconciler/pipelinerun/resources/apply.go

```diff
@@ -51,6 +54,12 @@ func ApplyParameters(p *v1beta1.PipelineSpec, pr *v1beta1.PipelineRun) *v1beta1.
 				}
 			} else {
 				for _, pattern := range patterns {
+					// array indexing for param is alpha feature
+					if cfg.FeatureFlags.EnableAPIFields == config.AlphaAPIFields {
+						for i := 0; i < len(p.Default.ArrayVal); i++ {
+							stringReplacements[fmt.Sprintf(pattern+"[%d]", p.Name, i)] = p.Default.ArrayVal[i]
```

Comment:
nit: pattern + "[%d]"

Reply:
Thanks! This is something I'm not aware of before

Reply:
That's weird - can you show me the errors?

Reply:
gofmt will remove these spaces. So if I add these it will fail the lint check
I think it is in this log.
https://storage.googleapis.com/tekton-prow/pr-logs/pull/tektoncd_pipeline/4855/pull-tekton-pipeline-build-tests/1524386665529872384/build-log.txt

Reply:
Oh! I guess that's the new formatting then. You can close this thread now

Reply:
Got it, thanks!